pkg/synopsysctl/cmd_root.go

```diff
@@ -53,7 +53,9 @@ var rootCmd = &cobra.Command{
 		}
 		log.SetLevel(lvl)
 		// Sets kubeconfig and initializes resource client libraries
-		setResourceClients()
+		if err := setResourceClients(); err != nil {
+			return err
```

Comment:
Returning an error here will display the `synospysctl --help` message. Consider logging the error and returning `nil` instead.   Also, returning an error will prevent synospysctl from running, thus if you are not connected to a cluster and get an error from creating kubeconfig/kubeclient, then synospsysctl will fail.